src/source/tile.js

```diff
@@ -10,6 +10,7 @@ import {CollisionBoxArray} from '../data/array_types.js';
 import Texture from '../render/texture.js';
 import browser from '../util/browser.js';
 import {Debug} from '../util/debug.js';
+import type VertexBuffer from '../gl/vertex_buffer.js';
```

Comment:
why is this needed?

Reply:
For [these two new buffer](https://github.com/mapbox/mapbox-gl-js/blob/77611fbfe803b3e6226d02ba90c29694b515c208/src/source/tile.js#L93_L94):

```
    globeGridBuffer: ?VertexBuffer;
    globePoleBuffer: ?VertexBuffer;
```